s3tests_boto3/functional/test_s3.py

```diff
@@ -6159,13 +6159,15 @@ def test_buckets_list_ctime():
     before = datetime.datetime.now(datetime.timezone.utc) - datetime.timedelta(days=1)
 
     client = get_client()
+    buckets = []
     for i in range(5):
-        client.create_bucket(Bucket=get_new_bucket_name())
+        buckets.append(client.create_bucket(Bucket=get_new_bucket_name()))
```

Comment:
this creates an array of create_bucket() response objects, not of bucket names. so the loop below filters out all bucket names and doesn't check any ctimes

Reply:
I've fixed this and created a new pull request, #565, as it looks like I don't have rights to update this pull request.